src/vdaf.rs

```diff
@@ -246,9 +253,36 @@ pub trait Aggregator<const VERIFY_KEY_SIZE: usize, const NONCE_SIZE: usize>: Vda
         input_share: &Self::InputShare,
     ) -> Result<(Self::PrepareState, Self::PrepareShare), VdafError>;
 
-    /// Preprocess a round of preparation shares into a single input to [`Aggregator::prepare_step`].
+    /// Preprocess a round of preparation shares into a single input to [`Self::prepare_next`].
+    ///
+    /// Implements `Vdaf.prep_shares_to_prep` from [VDAF].
+    ///
+    /// # Notes
+    ///
+    /// [`Self::prepare_shares_to_prepare_message`] is preferable since its name better matches the
+    /// specification.
+    ///
+    /// [VDAF]: https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-vdaf-07#section-5.2
+    #[deprecated(
```

Comment:
IMO, since this PR will be part of an effective-major-version bump, we may as well just drop the old methods. Backwards compatibility is not a concern, and if we keep these methods we'll need to keep them until the next effective-major-version bump.

Reply:
This way it's easier to give people a hint of what to move to. Filed #745 to remove them later.